TransbankPosSDK/Responses/RegisterCloseResponse.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Transbank.POS.Utils;
+
+namespace Transbank.POS.Responses
+{
+    public class RegisterCloseResponse
+    {
+        public LoadKeyCloseResponse Response {get;}
+
+        public int FunctionCode => Response.function;
+        public string Result => ResponseCodes.Map[Response.responseCode];
```

Comment:
just a question, Will `ResponseCodes.Map` never be empty?

Reply:
Yes, it convert the possible POS response codes to Text.